redeem/PathPlanner.py

```diff
@@ -52,9 +52,15 @@ def __init__(self):
         AlarmCallbackNative.__init__(self)
 
     def call(self, type, message, short_message):
-        endstop = PruInterface.get_endstop_triggered()
-        message += ": "+["unknown", "X1", "Y1", "Z1", "X2", "Y2", "Z2"][int(np.log2([endstop])[0]+1)]
-        logging.error("Native path planner alarm: {} {} {}".format(type, message, short_message))
+        if "soft endstop" in message.lower():
+            logging.error("{}, {}".format(message, short_message))
```

Comment:
I find that string comparison in these cases can be brittle. Maybe a future enhancement would be to create a canonical list (enum?) of alarm/error codes?